src/web/mjs/connectors/Manatoki.mjs

```diff
@@ -0,0 +1,52 @@
+import GnuBoard5BootstrapBasic2 from './templates/GnuBoard5BootstrapBasic2.mjs';
+
+export default class Manatoki extends GnuBoard5BootstrapBasic2 {
+
+    constructor() {
+        super();
+        super.id = 'manatoki82';
+        super.label = 'Manatoki';
+        this.tags = [ 'webtoon', 'korean' ];
+        this.url = 'https://manatoki82.net';
+
+        this.path = '/comic/p%PAGE%';
+        this.pathMatch = /comic\/p(\d+)/;
+        this.queryMangasPageCount = 'div.list-page ul.pagination li:last-child a';
+        this.queryMangas = 'ul#webtoon-list-all li div.img-item div.in-lable a';
+        this.queryChapter = 'div.serial-list li.list-item div.wr-subject a';
+        this.scriptPages = `
+        new Promise(resolve => {
```

Comment:
This did not work for me, it detected 0 images for https://manatoki82.net/comic/5896353 instead of expected 16.
This also does not filter out the (hidden) fake images that were added as protection against scrapers.

Reply:
Seems to work on https://manatoki82.net/comic/93125, should combine into a query that will work on both structures and correctly ignores the fake images

Reply:
there seems to be 2 different ways it serves images  
1needs div.view-padding div > img
1needs div.view-padding div > p:not([className]) img